features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumResourceScope.kt

```diff
@@ -182,10 +184,9 @@ internal class RumResourceScope(
         eventTime: Time,
         writer: DataWriter<Any>
     ) {
-        attributes.putAll(GlobalRumMonitor.get(sdkCore).getAttributes())
```

Comment:
I think this PR contains the changes from the other one ? I think you need a rebase here ?